packages/@aws-cdk/aws-pipes-alpha/awslint.json

```diff
@@ -0,0 +1,5 @@
+{
+  "exclude": [
+    "no-unused-type:@aws-cdk/aws-pipes-alpha.PipeVariable"
```

Comment:
I'm okay with this in principle, but why is not used?

Reply:
It's used only in the dame file as its declared. The DynamiInput class uses the enum as type. 
Depending on if we drop the enum entirely, the exclude can be reverted.

Reply:
Yes, you should be able to remove this.